api/v1beta1/cryostat_types.go

```diff
@@ -86,6 +86,25 @@ type CryostatSpec struct {
 	// +optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="JMX Connections Cache Options"
 	JmxCacheOptions *JmxCacheOptions `json:"jmxCacheOptions,omitempty"`
+	// Resource requirements for the Cryostat deployment
+	// +optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:resourceRequirements"}
+	Resources ResourceConfigList `json:"resources,omitempty"`
+}
+
+type ResourceConfigList struct {
+	// Resource requirements for the Cryostat application
```

Comment:
Hey John, would you mind amending this comment as follows:
```suggestion
	// Resource requirements for the Cryostat application. If specifying a memory limit, at least 768MiB is recommended.
```

We recently encountered some out-of-memory issues during testing, so this would help users. Once added, please re-run `make bundle`.